modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -242,4 +302,27 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test GetIsDefaultMethodValue {
+		task ("Given API Application created") {
+			ApplicationAPI.createAPIApplication(
+				baseURL = "test1",
+				title = "test1");
+		}
+
+		task ("When creating endpoint with valid data") {
+			APIBuilderUI.goToEditRelatedEntryInEditApplication(
```

Comment:
We need to create an endpoint for the api application

Reply:
Please remove the Given in this case. As shared, we should not update GWT, the original GWT is
> Given API Application created
> When creating endpoint with valid data
> Then enpoint method is GET by default
And we already create an api application with an endpoint in setup(), so in this case we only need to add
- When creating endpoint with valid data (in this we can create a new endpoint)
- Then enpoint method is GET by default (the AssertTextEquals you are using in line323 look perfect)
Hope this is more clear and please help update, thanks!